.github/actions/jx-updatebot-pr/action.yml

```diff
@@ -0,0 +1,111 @@
+# https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions
+name: 'jx-updatebot-pr'
+description: |
+  Create a Pull Request on each downstream repository
+  See: https://github.com/jenkins-x-plugins/jx-updatebot
+branding:
+  icon: 'git-pull-request'
+  color: 'blue'
+inputs:
+  auto-merge:
+    required: false
+    description: should we automatically merge if the PR pipeline is green (default true)
+    default: 'true'
+  version:
+    required: true
+    description: the version number to promote. If not specified uses $VERSION or the version file
```

Comment:
Not sure I follow the "if not specified" part since the parameter is required?

Reply:
Makes sense. I will update it